docker-compose-db.yml

```diff
@@ -0,0 +1,34 @@
+#postgres server is visible as pgsql-server in pgadmin panel
```

Comment:
Why did u create docker-compose?
This whole file is unnecesary.

We have one DB configured in project, and there i want to save a memes.

Reply:
I want to only build database and no other services for developement. This my personal preference i could delete this compose if You want.